eng/pipelines/templates/steps/build-artifacts.yml

```diff
@@ -40,6 +40,22 @@ steps:
       pip install wheel setuptools pathlib twine readme-renderer[md] packaging tox tox-monorepo
     displayName: 'Prep Environment'
 
+  - pwsh: |
```

Comment:
I see these _exact_ steps present in `archetype-sdk-client`. Is there anyway we can extract this to a `dev_build` step template and call it in both places?

Reply:
Moved duplicate code into set_dev_build.yml and referred as a template